src/storage/src/table/storage_table.rs

```diff
@@ -618,16 +618,14 @@ impl<S: StateStore, RS: RowSerde, const T: AccessType> StorageTableBase<S, RS, T
     }
 
     /// Construct a [`StorageTableIter`] for batch executors.
-    /// Differs from the streaming one, this iterator will wait for the epoch before iteration, and
-    /// the order of the rows among different virtual nodes is not guaranteed.
-    // TODO: introduce ordered batch iterator.
```

Comment:
@st1page Could you please confirm if it is okay, that we always return ordered values here, or should I create another method, for ordered batch_iter?